packages/frontend/core/src/types/dnd.ts

```diff
@@ -0,0 +1,55 @@
+import type { DNDData } from '@affine/component';
+
+export interface AffineDNDData extends DNDData {
```

Comment:
What if some node can only be dragged or dropped?

Reply:
I define them in an interface for easy use

eg.

```ts
useDropTarget<AffineDNDData>(() => ({
    data: {} // <----- AffineDNDData['dropTarget']
    onDrop(data) {
           // ^----- AffineDNDData['draggable']
    },
}))
```